Automattic/simplenote-macos
# Displaying Notes List on Search

### Fix
In this PR we're making sure the Notes List is visible, whenever the app enters into Search Mode.

cc @eshurakov (Thank you sir!!)
Ref. #626

### Test
1. Press **CMD + Shift + F**
2. Verify you've entered into Focus Mode
3. Press over the top right 🔍 icon

- [ ] Verify the Notes List is immediately revealed
- [ ] Verify the Tags List remains hidden

### Release
`RELEASE-NOTES.txt` was updated in dcc972d with:

> Entering Search Mode now displays automatically the Notes List #770
